### DISCUSSION TITLE
大于100M文件无法互相传送？

### DISCUSSION THREAD

**Discussion by willxing8** [localsend/localsend#591] [L-简体中文]
特别是视频文件，超过100M大小文件就会出现错误提示，请大佬们解决。使用最新版1.10无法解决这个问题，尝试使用1.7.0版本仍然存在这个问题。
![20230726170851](https://github.com/localsend/localsend/assets/78459218/d01cf1fd-35ae-41a1-83a7-96c82d1c0a33)
![20230726170853](https://github.com/localsend/localsend/assets/78459218/c9dc2ad4-2aa0-4e24-8e22-26e18a9511a0)

**Comment by mm10079** [comment]
第一張圖片是權限不足，你要檢查權限問題
